<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="menu_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>900</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>15</y>
         <width>376</width>
         <height>372</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="menu_verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="menu_verticalLayout" stretch="0,0,0,0,0,0,0,0,0">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QLabel" name="category_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Категория</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="category_comboBox">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="name_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Наименование</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="names_comboBox">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="article_number_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Артикул</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="article_number_lineEdit">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="maker_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Производитель</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="maker_comboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="amount_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Количество</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="amount_spinBox">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="amount_comboBox">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>170</width>
                <height>22</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="residue_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Остаток</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="residue_doubleSpinBox">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximum">
               <double>999999.989999999990687</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="residue_comboBox">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>170</width>
                <height>22</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="expiration_date_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Срок годности</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="expiration_date_dateEdit">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="location_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Местонахождение</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="location_comboBox">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="duplicatesEnabled">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="modelColumn">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="comment_input_label">
              <property name="minimumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>105</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Комментарии</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="comment_lineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>22</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="add_pushButton">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="search_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Поиск по:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="get_excel_pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>550</y>
         <width>120</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Сформировать Excel</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>460</y>
         <width>381</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="search_name_horizontalLayout">
        <item>
         <widget class="QLabel" name="search_name_label">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Названию</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_name_comboBox">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>490</y>
         <width>381</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="search_expiration_date_horizontalLayout">
        <item>
         <widget class="QLabel" name="search_expiration_date_label">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Сроку годности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_expiration_date_comboBox">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>520</y>
         <width>381</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="category_horizontalLayout">
        <item>
         <widget class="QLabel" name="search_category_label">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Категории</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_category_comboBox">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>243</width>
         <height>28</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="delete_horizontalLayout">
          <item>
           <widget class="QLabel" name="id_delete_label">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="delite_lineEdit">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_pushButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="tableView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AllEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
